orchagent/port/porthlpr.h

```diff
@@ -6,6 +6,22 @@
 #include <string>
 
 #include "portcnt.h"
+#include "portschema.h"
```

Comment:
@dgsudharsan the original intention of `portsorch` refactoring was to hide the properties parsing metadata from the mainline code. I do recommend to consider a different approach